src/main/webapp/sass/_core.scss

```diff
@@ -3076,10 +3076,102 @@ div.border {
 	font-style:italic;
 }
 
-#body .fullWidthAlert {
-	@include border-radius(0px);
-	box-shadow: none;
-	padding: 7px 15px;
+#body .synapseAlert {
+	&.global {
+		width: 100%;
+		position: fixed;
+		bottom: 0px;
+		left: 0px;		
+	}
+	padding: 14px;
+	z-index: 500;	
+	display: grid;
+	grid-template-columns: 50px auto 150px 200px;
+	column-gap: 10px;
+	align-items: center;
+	justify-items: start;
+	.iconArea {
+		grid-column: 1 / span 1;
+		grid-row: 1 / span 1;
+		min-width: 50px;
+	}
+	.messageArea {
+		max-width: 700px;
+		grid-row: 1 / span 1;
+	}
+	.iconForeground {
+		color: white;
+	}
+	@media (min-width: 992px) {
+		grid-template-rows: auto;
+		.messageArea {
+			grid-column: 2 / span 1;
+		}
+		.secondaryButton {
+			grid-column: 3 / span 1;
+			grid-row: 1 / span 1;
+			justify-self: end;
+		}
+		.primaryButton {
+			grid-column: 4 / span 1;
+			grid-row: 1 / span 1;
+			justify-self: end;
+		}
+	}
+	@media (max-width: 992px) {
+		grid-template-rows: auto 75px;
+		.messageArea {
+			grid-column: 2 / span 3;
+		}
+		.secondaryButton {
+			grid-column: 4 / span 1;
+			grid-row: 2 / span 1;
+		}
+		.primaryButton {
+			grid-column: 2 / span 2;
+			grid-row: 2 / span 1;
+		}
+	}	
+}
+
+// Bootstrap Alert overrides
+#body {
```

Comment:
override all alerts in synapse.org to have the new styles from the design (including the synapseAlerts that have special styling above).

Reply:
Just concerned that this might affect alerts created in SRC before those are modified to look like these alerts. Consider double checking e.g. the personal access tokens danger alert when you try to create a token with no name.

Reply:
<img width="592" alt="Screen Shot 2021-03-02 at 1 49 25 PM" src="https://user-images.githubusercontent.com/1864447/109719883-29998000-7b5e-11eb-8105-1514b8871463.png">

Reply:
Fantastic!